sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -891,9 +891,9 @@ class SubquerySuite extends QueryTest with SharedSparkSession {
 
         val sqlText =
           """
-            |SELECT * FROM t1
+            |SELECT * FROM t1 a
             |WHERE
-            |NOT EXISTS (SELECT * FROM t1)
+            |NOT EXISTS (SELECT * FROM t1 b WHERE a.i = b.i)
```

Comment:
Why need to change the existing test?

Reply:
> Why need to change the existing test?

https://github.com/apache/spark/pull/26437#discussion_r362742121